bigbluebutton-html5/imports/ui/components/video-provider/service.js

```diff
@@ -34,6 +34,32 @@ const {
 const TOKEN = '_';
 
 class VideoService {
+  static isUserPresenter(userId) {
+    const user = Users.findOne({ userId },
+      { fields: { presenter: 1 } });
+    return user ? user.presenter : false;
+  }
+
+  // Paginated streams: sort with following priority: local -> presenter -> alphabetic
```

Comment:
The definition stands for `local -> presenter -> alphabetic` but I can only notice `presenter -> alphabetic`. Is this intended?

Reply:
The local ordering is guaranteed in `getVideoPage`'s partitioning (mine, others).